search/search_quality/sample.cpp

```diff
@@ -74,6 +59,17 @@ void FromJSONObject(json_t * root, string const & field, m2::RectD & result)
   result.setMaxY(maxY);
 }
 
+void FromJSONObject(json_t * root, string const & field, m2::PointD & result)
```

Comment:
BTW, could you please move these quite special json helpers to the anonymous namespace?